### PR TITLE
Fixes732 - Add a link to etherscan contract in token list

### PR DESCRIPTION
**Pull request by personnamedmike**
https://github.com/airswap/airswap-web/assets/90465420/74817c79-48ae-4dd6-80bd-f8211619ec44

### CODE REVIEW

**Comment by personnamedmike** [comment]
Waiting on confirmation from design team that we're going with this design

**Comment by personnamedmike** [comment]
Fixed up styling and added tooltip

https://github.com/airswap/airswap-web/assets/90465420/ad51c71e-c8af-4652-a531-15bd39f350c2

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
     "token": "Token",
     "undefined": "undefined",
     "unknown": "Unknown",
+    "verifyEtherscan": "Verify on Etherscan",
```
Comment (dmosites) [inline public/locales/en/translation.json:66]: Each chain has a different block explorer, so how about just "Verify token"?

**Review by dmosites — CHANGES_REQUESTED** [review]

File: public/locales/en/translation.json (hunk 1 above)
Comment (personnamedmike) [reply public/locales/en/translation.json]: fixed

**Review by dmosites — APPROVED** [review]

**Pull request merged**